routes.py

```diff
@@ -8,5 +10,10 @@ def hello_world():
     return "<h1>Hello, World!</h1>"
 
 
+@app.route("/foo")
```

Comment:
Update to a more descriptive path.

Reply:
How about `/scramble` cuz this is where the scrambling algorithm will eventually output?

Reply:
I'd name it what it's currently doing. Let's keep the path in line with the functionality. We can always rename it or add another route later.

Reply:
Good thinking, thanks.

Reply:
Feel free to click the "Resolve conversation" button for this thread.